config/wdio/wdio.conf.js

```diff
@@ -0,0 +1,21 @@
+const wdioConf = require('terra-toolkit/config/wdio/wdio.conf');
+const webpackConfig = require('../webpack/webpack.config');
+
+const config = {
+  ...wdioConf.config,
+
+  webpackConfig,
+
```

Comment:
We need to disable axe's 'one-landmark-main' rule. See https://github.com/cerner/terra-toolkit/pull/88